### DISCUSSION TITLE
button clear multiselect

### DISCUSSION THREAD

**Discussion by pyinstallerbaggins** [marimo-team/marimo#2250] [Q&A] [answered]
How does one approach clearining a multiselect with a button? It does not clear selections.
![image](https://github.com/user-attachments/assets/ee5b8b7c-8d6d-4c7b-ad52-ff7b99510c62)

**Comment by akshayka** [comment, marked as answer]
You'll need to construct the multiselect and reference the button in the same cell

https://marimo.app/l/u0o290
